src/ScheduledJobs/JobHandlers/ParticipantOdpExportJobHandler.cs

```diff
@@ -12,7 +13,9 @@
 namespace ScheduledJobs.JobHandlers
 {
     // Reference class for handler generic type param
-    public class ParticipantOdpExport { }
+    public class ParticipantOdpExport
```

Comment:
What's this for?

Reply:
This was existing code, looks to be a common pattern across all the handlers, it is used to reference which handler is being called when HandleAsync is called as the naming convention must stay the same for the object type being passed in is what distinguishes it